app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,22 @@
  *= require_tree .
  *= require_self
  */
+
+:root {
+  --blue: #3778c2;
+  --green: #5fb523;
+  --grey: #434b54;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Montserrat", sans-serif;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Colors.-,Typography%3A%20font%20face,-%2C%20size%20and%20weight), Please use the correct font family which is `proxima nova` for all headlines text elements in your app, not `Montserrat` as you did.

![image](https://github.com/vikas924/budget-app/assets/68796232/26ee2b81-9de3-4562-bb7a-8db38c34afff)

Reply:
its paid font from adobe mostly others also using some other font and their project approved

Reply:
You can download it from [here](https://befonts.com/proxima-nova-font.html) then use it inside your app.